src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -1042,14 +1042,15 @@ public override long GetBytes(int ordinal, long dataOffset, byte[]? buffer, int
             if (buffer != null && (length < 0 || length > buffer.Length - bufferOffset))
                 throw new IndexOutOfRangeException($"length must be between {0} and {buffer.Length - bufferOffset}");
 
-            var fieldDescription = CheckRowAndGetField(ordinal);
-            var handler = fieldDescription.Handler;
+            var field = CheckRowAndGetField(ordinal);
+            var handler = field.Handler;
             if (!(handler is ByteaHandler))
-                throw new InvalidCastException("GetBytes() not supported for type " + fieldDescription.Name);
+                throw new InvalidCastException("GetBytes() not supported for type " + field.Name);
```

Comment:
Don't you want to move this to ThrowHelper too? Also, let's put the name in single quotes.

Reply:
As a separate pull request. We even have an issue for this.

Reply:
Not that I care, but what's the difference between moving this one and moving the other one (which this PR does)?

Reply:
We have a lot of exceptions to move, so I would like to do it a separate change (: